lib/wallet/channel_change_notifier.dart

```diff
@@ -2,15 +2,87 @@ import 'dart:async';
 
 import 'package:flutter/material.dart';
 import 'package:ten_ten_one/bridge_generated/bridge_definitions.dart';
+import 'package:ten_ten_one/cfd_trading/validation_error.dart';
 import 'package:ten_ten_one/ffi.io.dart' if (dart.library.html) 'ffi.web.dart';
+import 'package:ten_ten_one/wallet/action_card.dart';
+import 'package:ten_ten_one/wallet/open_channel.dart';
 
 class ChannelChangeNotifier extends ChangeNotifier {
-  ChannelState state = ChannelState.Unavailable;
+  ChannelState? state;
 
   bool isInitialising() => state == ChannelState.Establishing;
 
+  bool isDisconnected() => state == ChannelState.Disconnected;
+
   bool isAvailable() => state == ChannelState.Available;
 
+  bool isUnavailable() => state == ChannelState.Unavailable;
+
+  Message? validate() {
+    if (isUnavailable()) {
+      return Message(
+          title: 'No channel with 10101 maker',
+          details: 'You need an open channel with the 10101.',
+          type: AlertType.warning);
+    }
+
+    if (isInitialising()) {
+      return Message(
+          title: 'Channel not confirmed',
+          details: 'Please wait until your channel has 1 confirmation.',
+          type: AlertType.warning);
+    }
+
+    if (isDisconnected()) {
+      return Message(
+          title: 'Disconnected maker',
+          details: 'You are currently not connected to the maker.',
```

Comment:
If we want to avoid the "maker" terminology we might want to say

```suggestion
          title: 'Not connected to 10101',
```

The `details` section feels a bit redundant here.